### PR TITLE
profiles/logical/holo/hydra/master/specs: holo-envoy

### PR DESCRIPTION
**Pull request by samrose**
Adding holo-envoy project to hydra with jobsets and spec file

### CODE REVIEW

**Comment by samrose** [comment]
@mjbrisebois just FYI to flag you that this is the other piece of the puzzle, and I have created the project at https://hydra.holo.host/project/holo-envoy

File: profiles/logical/holo/hydra/master/specs/holo-envoy.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "checkinterval": 10,
+  "emailoverride": "",
+  "enabled": true,
+  "enableemail": false,
+  "hidden": false,
+  "inputs": {
+    "holo-nixpkgs": {
```
Comment (yegortimoshenko) [inline profiles/logical/holo/hydra/master/specs/holo-envoy.json]:
```suggestion
    "holo-envoy": {
```

File: profiles/logical/holo/hydra/master/specs/holo-envoy.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "emailresponsible": false,
+      "type": "git",
+      "value": "https://github.com/Holo-Host/holo-envoy.git develop"
+    },
+    "holo-nixpkgs-pull-requests": {
```
Comment (yegortimoshenko) [inline profiles/logical/holo/hydra/master/specs/holo-envoy.json]:
```suggestion
    "holo-envoy-pull-requests": {
```

File: profiles/logical/holo/hydra/master/specs/holo-envoy.json (hunk 1 above)
Comment (samrose) [reply profiles/logical/holo/hydra/master/specs/holo-envoy.json]: fixed

File: profiles/logical/holo/hydra/master/specs/holo-envoy.json (hunk 2 above)
Comment (samrose) [reply profiles/logical/holo/hydra/master/specs/holo-envoy.json]: fixed (thanks)

**Pull request merged**